REPL.cpp

```diff
@@ -116,12 +116,11 @@ bool REPL::ExecuteSwift(std::string line)
     m_invocation.getFrontendOptions().ModuleName = input.module_name.c_str();
     m_invocation.getIRGenOptions().ModuleName = input.module_name.c_str();
 
-    constexpr auto src_file_kind = swift::SourceFileKind::Main;
-    swift::SourceFile *src_file = new (*m_ast_ctx) swift::SourceFile(*module,
-                                                                     src_file_kind,
-                                                                     input.buffer_id,
-                                                                     implicit_import_kind,
-                                                                     false);
+    swift::SourceFile *src_file = new (*m_ast_ctx) swift::SourceFile(
+        *module, swift::SourceFileKind::Main, input.buffer_id,
+        implicit_import_kind, false);
```

Comment:
Small nit: It might be helpful to leave a small comment to indicate what the false is for. e.g.

`foo(/*is_bar*/ false);`

Reply:
Oh it looks like that's a default argument now - I'll just get rid of the false altogether.